en/snap/platform/overview.md

```diff
@@ -193,7 +193,18 @@ On **Sandbox** mode:
 Done! Now your Shopify online shop is ready to use Midtrans as payment gateway.
 ![shopify](./../../../asset/image/shopify-17.png ':size=400')
 
-<hr><br><br>
+<hr>
+
+### Unable to Install Specific Payment Methods
+In the rare case, Shopify can not install a specific payment method as well, to solve this case please follow the step below:
```

Comment:
> Shopify can not install a specific payment method as well, to solve this case

This wording is not accurate of the issue, please be more specific and accurate. Try these:
>In the rare case that you are not able to install specific payment methods, such as you are not presented with the install button on Step 2. Please try the following alternatives:

Reply:
make senses